### ISSUE TITLE
convert from camelcase to snakecase before casting

### ISSUE THREAD

**Issue by ericsullivan** [annkissam/common_graphql_client#18]
[potential library](https://github.com/johnnyji/proper_case)

**Comment by aditya7iyengar** [comment]
I don't know if we should dictate this in common graphql client. I made `resolve_resolve` overridable, so people can choose how to interpret the results.

**Comment by ericsullivan** [comment]
Fair enough. Want to drop a code snippet in here (and example of overriding) and close this?
